azure-pipelines.yml

```diff
@@ -1,30 +1,54 @@
-# .NET Desktop
-# Build and run tests for .NET Desktop or Windows classic desktop solutions.
-# Add steps that publish symbols, save build artifacts, and more:
-# https://docs.microsoft.com/vsts/pipelines/apps/windows/dot-net
+# ASP.NET Core
```

Comment:
this should just be .net core

Reply:
Ah.  That should be .NET Standard.  .NET Standard is the API surface;  we want this on .NET Framework (desktop) and .NET Core.